docs/api/crash-reporter.md

```diff
@@ -27,6 +27,7 @@ following projects:
 Or use a 3rd party hosted solution:
 
 * [Backtrace I/O](https://backtrace.io/electron/)
+* [Sentry](https://sentry.io/welcome/) ([doc](https://docs.sentry.io/clients/electron/))
```

Comment:
I would suggest a single link to the doc:

```
[Sentry](https://docs.sentry.io/clients/electron)
```

Reply:
Done, thanks.